plugins/network/modem-nvg510

```diff
@@ -0,0 +1,178 @@
+#! /usr/bin/env perl
+
+
+=head1 NAME
+
+modem-nvg510 - Plugin to monitor Motorola/Arris NVG510 DSL modem stats
+
+=head1 CONFIGURATION
+
+	[modem-nvg510]
+	env.url http://192.168.1.254/cgi-bin/dslstatistics.ha
+
+=head1 AUTHOR
+
+Samuel Smith <esaym (snail) cpan.org>
+
+=cut
+
+use strict;
+use warnings;
+use HTTP::Tiny;
+
+
+
+use constant {
+	down_rate => 0,
+	up_rate => 1,
+	sn_down => 14,
+	sn_up => 15,
+	line_attn_down => 16,
+	line_attn_up => 17,
+	power_down => 18,
+	power_up => 19,
+	err_sec_down => 20,
+	err_sec_up => 21,
+	los_down => 22,
+	los_up => 23,
+	lof_down => 24,
+	lof_up => 25,
+	fec_down => 26,
+	fec_up => 27,
+	crc_down => 28,
+	crc_up => 29,
+};
+
+if(defined $ARGV[0] and $ARGV[0] eq 'autoconf'){
```

Comment:
It should have a magic marker (#%# family=auto) in the comments somewhere for munin-node-configure to call it with "autoconf"

See example at https://github.com/munin-monitoring/munin/blob/devel/plugins/node.d/apache#L99